src/components/nav/components/searchBox/SearchBox.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { useHandleStatus } from "../../../../hooks/useHandleStatus";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faMagnifyingGlass } from "@fortawesome/free-solid-svg-icons";
+import { faAngleUp } from "@fortawesome/free-solid-svg-icons";
+
+const SearchBox = () => {
+  const { isStatus, handleStatus } = useHandleStatus();
+
+  const [selectStatus, setSelectStatus] = useState("지역");
+
+  const innerSelectorName = event => {
+    const { innerText } = event.target;
+    setSelectStatus(innerText);
+    handleStatus();
+  };
+
+  const innerPlaceHolder = () => {
+    if (selectStatus === "지역") {
+      return "동,읍,면을 입력해 주세요";
+    } else if (selectStatus === "지하철") {
+      return "지하철역 이름을 입력해 주세요";
+    } else if (selectStatus === "대학교") {
+      return "대학교 이름을 입력해 주세요";
+    }
+  };
+  return (
+    <Wrapper>
+      <SelectWrapper>
+        <Selection onClick={handleStatus}>
+          <SelectionOption>
+            <Option>{selectStatus}</Option>
+          </SelectionOption>
+          {isStatus ? (
+            <FontAwesomeIcon
+              icon={faAngleUp}
+              style={{
+                position: "absolute",
+                top: "-2px",
+                right: "20px",
+                transform: "translate(50%, 50%)",
+              }}
+            />
+          ) : (
+            <FontAwesomeIcon
+              icon={faAngleUp}
+              style={{
+                position: "absolute",
+                top: "-2px",
+                right: "20px",
+                transform: "translate(50%, 50%) rotateX(180deg)",
+              }}
+            />
+          )}
+        </Selection>
+        {isStatus && (
+          <SelectModal>
+            <TitleOption onClick={innerSelectorName}>지역</TitleOption>
+            <TitleOption onClick={innerSelectorName}>지하철</TitleOption>
+            <TitleOption onClick={innerSelectorName}>대학교</TitleOption>
+          </SelectModal>
+        )}
+      </SelectWrapper>
+
+      <SearchWrapper>
+        <Search type="text" placeholder={innerPlaceHolder()} />
```

Comment:
객체로 만들어 사용한다면 다음처럼 사용할 수 있습니다.
그리고 state 명칭이 좀 더 명확하면 좋겠네요!
```suggestion
        <Search type="text" placeholder={LOCATION_DESC[selectStatus]} />
```

Reply:
넵 해당 부분 수정하고 state 명칭을 selectCategory라고 지었습니다